_build/templates/default/sass/index.scss

```
@@ -1456,6 +1456,7 @@ iframe[classname="x-hidden"] {
       .icon {
         text-align: center;
         margin-left: 10px;
+        width: 18px;
```

```suggestion
        @extend .#{$fa-css-prefix}-fw;
```

To prevent unnecessary changes down the road It would be better to use the fixed-width utility FontAwesome provides. If my suggestion is approved you can also remove the `text-align: center;`.

---

Thats better!